dev/buildtool/changelog_commands.py

```diff
@@ -304,7 +304,7 @@ def __init__(self, factory, options, **kwargs):
             with open(options.relative_to_bom_path, encoding="utf-8") as stream:
                 self.__relative_bom = yaml.safe_load(stream.read())
         elif options.relative_to_bom_version:
-            self.__relative_bom = HalRunner(options).retrieve_bom_version(
+            self.__relative_bom = BomSourceCodeManager.bom_from_gcs_bucket(
```

Comment:
Perhaps out-of-scope for this PR, but is there logic in buildtool to do the dance of next-release-preview for .0 versions?

Reply:
If not, could you add something to the readme for it?

Reply:
No logic in this PR for that.

README had note, added call out in the new section - https://github.com/spinnaker/buildtool/pull/196/commits/996e11f8a726d99bb37bca5ed71be36f1147ba7c